envoy/formatter/substitution_formatter_base.h

```diff
@@ -86,9 +109,16 @@ template <class FormatterContext> class CommandParserBase {
 
 template <class FormatterContext>
 using CommandParserBasePtr = std::unique_ptr<CommandParserBase<FormatterContext>>;
-
-template <class FormatterContext>
-using CommandParsersBase = std::vector<CommandParserBasePtr<FormatterContext>>;
+using StreamInfoCommandParser = CommandParserBase<void>;
+using StreamInfoCommandParserPtr = CommandParserBasePtr<void>;
+
+template <class FormatterContext> constexpr absl::string_view formatterContextCategory() {
```

Comment:
comments?

Reply:
got it.

Reply:
I remove this function finally. This function is used to get short category name of the formatter context. And because `void` is used as type placeholder of stream info only providers, we need this function as a proxy to provide special support to the void type.

But in the latest commit, I create a type `StreamInfoOnlyFormatterContext` as type placeholder of stream info only providers to instead of `void`.
So, this function is unnecessary now. We can get the short category name by T::category() now for all types of formatter context.